### PR TITLE
Ft/data model

### PR DESCRIPTION
**Pull request by Bateyjosue**
# FEATURE DESCRIPTION

In this project, I use migration files to build the database schema.

### Cheers

# Happy Review

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: Linters
+
+on: pull_request
```
Comment (HAMMAS-SALEEM) [inline .github/workflows/linters.yml:3]: - [ ] Your linters are not passing all the test. Kindly fix your linter errors 😊

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Submission 🔴

## Hey @Bateyjosue :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following some requirements

Good job on implementing some of the requirements, but i cannot proceed review on your pull request, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml (hunk 1 above)
Comment (Bateyjosue) [reply .github/workflows/linters.yml:3]: @HAMMAS-SALEEM Bro. Hope you are fine. if you have checked the error you will find that it's a ruby version error

Comment (Bateyjosue) [reply .github/workflows/linters.yml:3]: Would you suggest me how to fix it, please? because I'm not getting the errors on my PC. Its only on github

Comment (HAMMAS-SALEEM) [reply .github/workflows/linters.yml:3]: Hey @Bateyjosue sorry for the late response.I can see you have already fixed the errors :blush: That's really a great thing which you did. Keep doing great things like that. Happy Coding :+1:

**Comment by arslanbisharat** [comment]
Hi @Bateyjosue, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has node_modules. It is not allowing me to review it properly. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @Bateyjosue ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️

## Optional suggestions

- [ ] Kindly fix the broken issues link to make your readme file more professional.
- [ ] You may also remove the video demo from your readme file since it is for a different project. This project doesn't require recording a video at the moment.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**